src/docs/reference/modules/Forms/README.md

```diff
@@ -21,6 +21,26 @@ Creating a form typically involves the following steps:
 3. Add the Form widget.
 4. Because the Form widget has the `FlowPart` attached, you can add other widgets to the Form widget. Add widgets such as Input, Textarea and Button to build up your form.
 
+### Configuring Form Visibility
+
+Once your form’s basic structure is in place, you can enhance it with conditional visibility rules:
```

Comment:
Please add an example for a high-level use-case at the end where this is useful. Like on a contact form, you can X.

```suggestion
Once your form’s basic structure is in place, you can enhance it with conditional visibility rules. These allow you to hide or show certain failed based on the values of other fields. For example, ...
```